src/Servers/Kestrel/Transport.Sockets/src/SocketConnectionListener.cs

```diff
@@ -66,9 +62,16 @@ internal void Bind()
                 throw new InvalidOperationException(SocketsStrings.TransportAlreadyBound);
             }
 
-            // TODO: Add support for UnixDomainSocket
+            Socket listenSocket;
```

Comment:
Maybe like this
```c#
Socket listenSocket = new Socket(EndPoint.AddressFamily, SocketType.Stream, EndPoint is UnixDomainSocketEndPoint ? ProtocolType.Unspecified : ProtocolType.Tcp);
```
?

Reply:
Yep, I was thinking about that after I saw this PR 😄